components/scream/src/share/io/scream_output_manager.cpp

```diff
@@ -98,16 +98,16 @@ setup (const ekat::Comm& io_comm, const ekat::ParameterList& params,
       // Output control
       auto& pl = m_params.sublist("Checkpoint Control");
       m_checkpoint_control.frequency  = pl.get<int>("Frequency");
-      m_checkpoint_control.frequency_units = pl.get<std::string>("Frequency Units");
+      m_checkpoint_control.frequency_units = pl.get<std::string>("frequency_units");
       m_checkpoint_control.nsteps_since_last_write = 0;
 
       // File specs
       m_checkpoint_file_specs.max_snapshots_in_file = 1;
       m_checkpoint_file_specs.num_snapshots_in_file = 0;
       m_checkpoint_file_specs.filename_with_time_string = pl.get("Timestamp in Filename",true);
       m_checkpoint_file_specs.filename_with_mpiranks    = pl.get("MPI Ranks in Filename",false);
-      m_checkpoint_file_specs.filename_with_avg_type    = pl.get("AVG Type in Filename",true);
-      m_checkpoint_file_specs.filename_with_frequency   = pl.get("Frequency in Filename",true);
+      m_checkpoint_file_specs.filename_with_avg_type    = pl.get("avg_type_in_filename",true);
```

Comment:
@jgfouca and @bartgol , did we also want to change the two parameters above this one?  Timestep in Filenam and MPI Ranks in Filename?

Reply:
@AaronDonahue , those parameters aren't in `cime_config/namelist_defaults_scream.xml` so they weren't processed by my script.